### PR TITLE
Black formatting + config

### PR DESCRIPTION
**Pull request by einarf**
Manually applied black formatting to most of the codebase manually inspecting and testing the changes,

A basic black config was also added in `pyproject.toml` setting line width to 120 and excluding several directories.

### CODE REVIEW

File: arcade/camera/grips/strafe.py [hunk 1]
```diff
@@ -18,8 +18,4 @@ def strafe(data: CameraData, direction: Tuple[float, float]) -> Tuple[float, flo
     _pos = data.position
 
     offset = _right * direction[0] + _up * direction[1]
-    return (
-        _pos[0] + offset[0],
-        _pos[1] + offset[1],
-        _pos[2] + offset[2]
-    )
+    return (_pos[0] + offset[0], _pos[1] + offset[1], _pos[2] + offset[2])
```
Comment (pushfoo) [inline arcade/camera/grips/strafe.py]: This is separation is kind of meaningful until this file is cleaned up more.
```suggestion
# fmt: off
return (
        _pos[0] + offset[0],
        _pos[1] + offset[1],
        _pos[2] + offset[2]
    )
# fmt: on
```

File: arcade/experimental/clock/clock_window.py [hunk 2]
```diff
@@ -777,16 +780,18 @@ def show_view(self, new_view: AnyView):
         """
         if not isinstance(new_view, View):
             raise TypeError(
-                f"Window.show_view() takes an arcade.View,"
-                f"but it got a {type(new_view)}.")
+                f"Window.show_view() takes an arcade.View," f"but it got a {type(new_view)}."
```
Comment (pushfoo) [inline arcade/experimental/clock/clock_window.py]: If we have one line, then no need for two separate f-strings. This should be merged into one.
```suggestion
                f"Window.show_view() takes an arcade.View, but it got a {type(new_view)}."
```

**Review by pushfoo — CHANGES_REQUESTED** [review]
Initial look done on subfolders up to those starting with e.

File: arcade/camera/grips/strafe.py (hunk 1 above)
Comment (einarf) [reply arcade/camera/grips/strafe.py]: This one is pretty straight forward to read in both forms imho. Subjective. I'll just commit the suggestion.

**Pull request merged**